snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/FileBasedSnapshotStore.java

```diff
@@ -471,6 +473,7 @@ private void moveToSnapshotDirectory(final Path directory, final Path destinatio
           destination,
           e);
     } catch (final IOException e) {
+      LOGGER.warn("Failed to move snapshot directory {} to {}", directory, destination, e);
```

Comment:
We already log the error after it bubbles up, no? What does it look like now if a failure occurs here? I'm not sure it's worth logging if we rethrow the exception.